doc/api/cli.md

```diff
@@ -298,6 +298,17 @@ added:
 
 Expose the [Web Crypto API][] on the global scope.
 
+### `--experimental-import=module`
```

Comment:
I don't know if you missed https://github.com/nodejs/node/pull/43942#issuecomment-1192942755, but it looks like the consensus is to go with `--import` rather than `--experimental-import`.
```suggestion
### `--import=module`
```

Reply:
yeah will do that as well